services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_api/_core.py

```diff
@@ -74,7 +69,7 @@ async def create_network(network_config: dict[str, Any]) -> NetworkId:
             network_name = network_config["Name"]
             # make sure the current error being trapped is network dose not exit
             if f"network with name {network_name} already exists" not in str(e):
-                raise e
+                raise
```

Comment:
Question: why this change? is it not better to `raise e`?

Reply:
see [https://beta.ruff.rs/docs/rules/verbose-raise/](https://beta.ruff.rs/docs/rules/verbose-raise/)